tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -76,6 +79,19 @@ def __init__(self, logger, binary, webdriver_binary="chromedriver",
                                          binary=webdriver_binary,
                                          args=webdriver_args)
 
+    def _adb_run(self, args):
+        self.logger.info('adb ' + ' '.join(args))
+        subprocess.check_call(['adb'] + args)
+
+    def setup(self):
+        self._adb_run(['wait-for-device'])
+        self._adb_run(['forward', '--remove-all'])
+        self._adb_run(['reverse', '--remove-all'])
+        # TODO: Use ports from config.
```

Comment:
Some logging confirms that there are in `server_config` in `executor_kwargs`, and I'm guessing the `executor_kwargs` return value ends up somehow propagated to the `ChromeAndroidBrowser` instance? In other words, is this easy to fix already? If so let's fix it, if it's hard because of some mis-layering, maybe file an issue about that layering problem.

Reply:
No, it doesn't. Although there is a hook for Browser modules to modify `executor_kwargs`, but the dictionary is not passed back to Browser classes, as Browsers are not Executors... I'm going to store the `server_config` in a module-local variable. It's not ideal, but probably better than hardcoding.